### ISSUE TITLE
Syncing not working as expected

### ISSUE THREAD

**Issue by Hans-Maulwurf** [tryunearth/app#8]
First: A automatic syncing would be great, at the moment I have to sync by pressing "sync reddit saves", haven't I?

In the description it says, the reddit limit of 1000 is expanded, so there is no limit in unearth. But when I clicked the sync button today, I saw 
![image](https://user-images.githubusercontent.com/9821544/84126384-1f6aab00-aa3e-11ea-836f-fd6650b6bfea.png) But I did not remove any saved items.

**Comment by Hans-Maulwurf** [comment]
No, I had my years-filled saved items and synced them. The screenshot was on the 2nd day of using unearth. Or do I not have to hit this button and it syncs automatic in the background some time?

**Comment by cedricium** [comment]
Syncing at this time is *not* automatic (though it is something I'd like to get around to implementing).

With that said, if you save anything on Reddit (or conversely, remove any saves), Unearth's snapshot of your saves will be lagging. Thus when you visit Unearth, the sync button is there to achieve parity between Reddit and Unearth—the idea being you're using the latest data when searching or organizing your saved content.

Since you mentioned you did not remove any saved items, I'm willing to bet this is an issue with the syncing mechanism. Will mark this as a bug and look into it right away.
